packages/vuetify/src/components/VCombobox/VCombobox.tsx

```diff
@@ -342,15 +341,17 @@ export const VCombobox = genericComponent<new <
         vTextFieldRef.value?.focus()
       }
     }
-    function select (item: ListItem) {
+    function select (item: ListItem, enterKey: Boolean = false) {
```

Comment:
```suggestion
    function select (item: ListItem, isEnterKey = false) {
```

you don't need to add type for the default value